doc/source/contributor.rst

```diff
@@ -0,0 +1,222 @@
+.. _developerdocumentation:
+
+=======================
+Developer Documentation
+=======================
+
+In this section, you will find documentation relevant to developing
+socok8s.
+
+.. _contributing:
+
+Contributor Guidelines
+======================
+
+Submodules
+----------
+
+This repository uses submodules. All of the following guidelines apply
+to this socok8s repository only. Please check other projects' practices
+before contributing to them, if a change needs to happen there.
+
+Before submitting code
+----------------------
+
+This is a very fast moving project, and you should probably contact us
+before starting to work on it.
+
+If you're willing to submit code, please remember the following rules:
+
+* All code should match our
+  :ref:`codeguidelines`.
+* All code requires to go through our :ref:`reviews`.
+* Documentation should be provided with the
+  code directly. See also :ref:`documentation`.
+
+.. _bug_reporting:
+
+Bug reporting process
+---------------------
+
+Bugs should be filed as Github issues.
```

Comment:
Also, i understand that socok8s is currently TP, but if and when we move to some official product, Bugzilla should be main point of reference

Reply:
See also community discussion vs not. If we want to build a community around the product, then it would depend on community usage of Bugzilla. Bugzilla seems fine to me, while less convenient than github issues.

Reply:
Bugzilla is open for community. I do not really have a problem with GitHub issues, it seems like a faste way to get feedback. However if we're gonna ship this as part of our product, we need some internal issue tracking.

Reply:
Agreed.